utils/time.js

```diff
@@ -0,0 +1,5 @@
+const getUnixEpochTime = (time = Date.now()) => {
+  return Math.floor(time / 1000)
+}
+
+module.exports = { getUnixEpochTime }
```

Comment:
Can you add JSDoc for this function?

Reply:
Do we have some JSDocs available in our project structure?
I couldn't find any.

If not, where should I create it?

Reply:
Existing code is having JSDoc annotations
Check this link and it'll all make sense 😉 https://jsdoc.app/about-getting-started.html#adding-documentation-comments-to-your-code

Reply:
I feel stupid now.😛

Pushed in new changes.